### DISCUSSION TITLE
Confirmation tap action to mirror that of normal button confirmations

### DISCUSSION THREAD

**Discussion by StuartHaire** [Clooos/Bubble-Card#303] [Feature requests]
Hi, 

I have started using your bubble card and it's great! I'm migrating a few things I have to use this as it's a much nicer way.
There is a small thing I was hoping could be changed..

When adding in confirmation to tap action eg:

```
    tap_action:
      action: toggle
      confirmation:
        text: This will toggle X do you want to proceed?
      service: script.myscript
```

This is displayed:
![2023-11-08T15 25 43](https://github.com/Clooos/Bubble-Card/assets/38922184/523337c1-7827-4417-ad77-b2eaf476ed2d)

However using the standard button card within HA and using confirmation displays this:
![2023-11-08T15 29 16](https://github.com/Clooos/Bubble-Card/assets/38922184/47834462-5659-44dd-9b53-664a3e6acd1e)

Basically it looks nicer with the "Are you sure?" heading rather than the website url as the heading and was wondering could this be modified to reflect the "Are you sure?" heading
